blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -439,7 +443,7 @@ export const actions = {
                                 return { ...data,
                                     id: event.jenkins_object_id, // make sure the runId is set so we can find it later
                                     state: 'RUNNING', // This is a horrible hack due to issues with QUEUED status
-                                    result: event.job_run_status,
+                                    result: 'UNKNOWN',
```

Comment:
We aren't using the state from the result because it often says `QUEUED` when we want `RUNNING`. The pipeline job moves from the queue, but isn't currently executing anything and still returns `QUEUED`. But we never get further updates when it actually *is* `RUNNING`.